src/test/java/org/javarosa/core/test/Scenario.java

```diff
@@ -273,6 +273,7 @@ public Scenario serializeAndDeserializeForm() throws IOException, Deserializatio
         );
 
         delete(tempFile);
+        deserializedFormDef.initialize(false, new InstanceInitializationFactory());
```

Comment:
This call is intent on haunting me: https://github.com/getodk/collect/pull/4367/files#diff-3cd25b78cc06d80783f3578f5ab3d7f33d62b3e726a86dcfddc4122fd9c63357R27

I think it's telling that we keep forgetting it in test code. It feels like some higher-level abstraction should be responsible for keeping track of/determining whether an instance is new and initializing the form def. Something for another time.